x-pack/plugins/security_solution/public/common/components/event_details/table/prevalence_cell.tsx

```diff
@@ -23,7 +24,7 @@ const PrevalenceCell = React.memo<AlertSummaryRow['description']>(
     if (loading) {
       return <EuiLoadingSpinner />;
     } else if (error) {
-      return null;
+      return <>{getEmptyValue()}</>;
```

Comment:
Maybe better to use `getEmptyTagValue` or `getEmptyStringTag`? But in general that is more than weird how many duplications we have :)

Reply:
Sure, works either way I guess 👍 (https://github.com/elastic/kibana/pull/128295/commits/e2f321f1fd6594acf54f782e7027b4558a5b681f)